src/components/molecules/Alert/Alert.stories.tsx

```diff
@@ -42,6 +42,13 @@ Success.args = {
     severity: AlertSeverity.Success,
 };
 
+export const Basic = Template.bind({});
```

Comment:
can you fix the stories in this PR for only 2 break points?

Reply:
Will do